### PR TITLE
Add example GPU workload

### PR DESCRIPTION
**Pull request by NathanTP**
Now that our fork of open-lambda has GPU support on master, I wanted to include some documentation on how to use it with SRK.

### CODE REVIEW

File: docs/source/Examples/GPU.rst [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+	$ ./srk function create -s examples/gpu
+	$ ./srk bench -b one-shot -a '{"test-size":1048576}' -n gpu
+
+Once again, you should see "Success" printed as the response from the function.
+
+You are now ready to start experimenting with GPU-enabled functions. You should
+be able to do anything with the GPU in the function that you could do locally.
+Keep in mind that OpenLambda limits the available function concurrency to the
+number of GPUs when gpu mode is enabled to avoid collisions with devices. This
```
Comment (jssmith) [inline docs/source/Examples/GPU.rst]: "collisions with devices" sounds a little strange. Is there another way to say this?

File: examples/gpu/README.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
+This example shows how to use CUDA with openlambda.
+
+# Overview
+The program does a simply vector add of two generated arrays and verifies that
```
Comment (jssmith) [inline examples/gpu/README.md]: Grammar: "does a simply"

File: examples/gpu/README.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+the add was done correctly. The size of the vectors to add is passed in as an
+argument to the cloud function.
+
+The main included source files are:
+  - f.py: the main cloud function called by openlambda
+  - vadd.cu: A generic vector-addition test (basically a hello world)
+  - vadd: The compiled standalone binary for vadd.cu. You can call this from the command line.
+  - vadd.so: The shared-library version of vadd.cu. This can be linked into other programs and called.
+
+f.py dynamically loads vadd.so using the ctypes library. It then converts it's
```
Comment (jssmith) [inline examples/gpu/README.md]: Gramar: "it's"

File: examples/gpu/README.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+argument (a dictionary generated from the JSON arguments passed to lambda) to
+native c types and invokes the cudaTest function.
+
+# Requirements
+This example requires a custom version of OpenLambda that supports GPUs:
+
+    $ git clone -b gpu git@github.com:NathanTP/open-lambda.git
+
+You can follow the instructions in that repo to install. You'll also need to
+configure srk to use that version (see srk/README.md for more details).
```
Comment (jssmith) [inline examples/gpu/README.md]: capitalize srk

**Comment by NathanTP** [comment]
Fixed all the feedback.

**Pull request merged**